### ISSUE TITLE
InvalidTypeFormat false positives

### ISSUE THREAD

**Issue by boesing** [webimpress/coding-standard#125]
Hey there,

I've found false positives regarding `WebimpressCodingStandard.Commenting.TagWithType.InvalidTypeFormat`

```php

class Foo
{
    /** @var array<string,mixed> */
    public $baz = [];
}
```

Expected: all good
Actual:
```
 Invalid type format with tag @var (WebimpressCodingStandard.Commenting.TagWithType.InvalidTypeFormat)
```

**Comment by boesing** [comment]
This also affects `WebimpressCodingStandard.Commenting.TagWithType.InvalidParamName` when written like this:


```php
class Bar
{
    /**
     * @param array<string, mixed> $config
     */
     public function __construct(array $config)
     {
     }
}
```

**Comment by michalbundyra** [comment]
@boesing thanks for reporting, this is something is not supported yet in webimpress coding-standard: generic types. I was planning to do it for next minor release, but development slowed down here. I am aware about this issue, but also I've seen many projects/people who were not using generics as PHPStorm doesn't support these correctly. I think the support is going to be added in next PHPStorm release so we definitely have to do it also here.
